cms/grading/tasktypes/Batch.py

```diff
@@ -262,6 +265,7 @@ def evaluate(self, job, file_cacher):
             commands,
             job.time_limit,
             job.memory_limit,
+            max_processes,
```

Comment:
I agree that it's not super clean, but I feel like `d[a] if a in d else d['default']` is not nice either, it's pretty much trying to reinvent d.get.
I could just get rid of using a separate variable and have `max_processes = LANGUAGE_TO_MAX_PROCCESSORS.get(language, LANGUAGE_TO_MAX_PROCCESSORS['default'])` instead, would that be better?
re `max_processes=num_processes`, I specifically *didn't* use keyword arguments because the other arguments didn't and it was obvious what its purpose was, is there a reason that I should here?